tests/constants.py

```diff
@@ -0,0 +1,4 @@
+BID = 0
+ASK = 1
+LONG = 0
+SHORT = 1
```

Comment:
Can be done later (or never) but there may be value in throwing YES/NO in here as well since we use them a lot.